src/utils.jl

```diff
@@ -727,7 +727,7 @@ function Base.StackTraces.StackFrame(frame::Frame)
     scope = scopeof(frame)
     if scope isa Method
         method = scope
-        method_args = something.(frame.framedata.locals[1:method.nargs])
+        method_args = something.(frame.framedata.locals[1:method.nargs]) :: Vector{Any}
```

Comment:
```suggestion
        method_args = [something(x) for x in frame.framedata.locals[1:method.nargs]]
```

maybe also works to make it understand the result is a `Vector`?

Reply:
That works too, yeah: done.

Also noticed that `fname` further down didn't get inferred as `Symbol`, so fixed that too - I suppose it couldn't hurt.